lib/Conversion/ExportVerilog/PruneZeroValuedLogic.cpp

```diff
@@ -109,13 +170,19 @@ void ExportVerilog::pruneZeroValuedLogic(hw::HWModuleOp module) {
   PruneTypeConverter typeConverter;
 
   target.addLegalDialect<sv::SVDialect, comb::CombDialect, hw::HWDialect>();
-
-  // Generic conversion and legalization patterns for operations that we
-  // expect to be using i0 valued logic.
   addPruningPattern<NoI0OperandPruningPattern<sv::PAssignOp>,
                     NoI0OperandPruningPattern<sv::BPAssignOp>,
                     NoI0OperandPruningPattern<sv::AssignOp>>(target, patterns,
                                                              typeConverter);
 
+  addNoI0ResultPruningPattern<
+      // SV ops
+      sv::WireOp, sv::RegOp, sv::ReadInOutOp,
+      // Prune all zero-width combinational logic.
+      comb::AddOp, comb::AndOp, comb::ConcatOp, comb::DivSOp, comb::DivUOp,
+      comb::ExtractOp, comb::ModSOp, comb::ModUOp, comb::MulOp, comb::MuxOp,
+      comb::OrOp, comb::ParityOp, comb::ReplicateOp, comb::ShlOp, comb::ShrSOp,
```

Comment:
Super nit: I think comb::ParityOp has always an i1 result.

Reply:
But it also makes sense to me that we register all combinatorial operations here for maintainability.